### DISCUSSION TITLE
Image Zooming

### DISCUSSION THREAD

**Discussion by mgurg** [jpanther/congo#553] [Q&A]
Is it possible to zoom in (enlarge to full screen) on an image after clicking on it?

Example from random website (not a Congo template): https://pietrasiak.com/one-line-of-code-that-did-cost-dollar8000
![Zrzut ekranu z 2023-05-08 12-27-39](https://user-images.githubusercontent.com/34517378/236801191-360cabbf-c251-4c98-903e-06094dcdf65b.png)

**Comment by Tanddant** [comment]
Looking at the telemetry of my blog, I agree!

21.13% of ~100 readers on my post [here](https://blog.dan-toft.dk/2023/03/march-2023-m365/) had dead clicks, most of which appear to be on images - it would be a very nice feature 😊

**Comment by DejavuMoe** [comment]
@jpanther Maybe using [medium-zoom](https://github.com/francoischalifour/medium-zoom) would be a good choice🤔

**Comment by stereobooster** [comment]
Maybe https://tailwindcomponents.com/component/alpinejs-tailwindcss-lightbox-modal

**Comment by stereobooster** [comment]
Another option https://github.com/stereobooster/svg-pan-zoom-gesture. It was created for SVGs in mind, but would generally work for any HTML element, including images

**Comment by yarikoptic** [comment]
FWIW, a candidate to consider for zooming (with a demo for a PR elsewhere to try on) is at 
- https://github.com/jpanther/congo/pull/1156
